.circleci/config.yml

```diff
@@ -213,7 +213,8 @@ jobs:
         name: Upload .app to sauce labs
         command: |
           source bin/sauce-pre-upload.sh
-          curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" -X POST -H "Content-Type: application/octet-stream" https://saucelabs.com/rest/v1/storage/automattic/Gutenberg-$SAUCE_FILENAME.app.zip?overwrite=true --data-binary @./gutenberg/packages/react-native-editor/ios/GutenbergDemo.app.zip
+          HOMEBREW_NO_AUTO_UPDATE=1 brew install curl
```

Comment:
Good idea @ceyhun! I verified that while downloading, the curl version is visible in the homebrew install log. Still, may I suggest doing a `/usr/local/opt/curl/bin/curl --version` after this so we can be sure what version is used? Not a blocker for this PR though. I'll 👍 it and you can merge or update if you want, thanks!

Reply:
Sounds good, added the --version command 👍